airflow/models/dagrun.py

```diff
@@ -268,10 +290,14 @@ def update_state(self, session=None):
         Determines the overall state of the DagRun based on the state
         of its TaskInstances.
 
-        :return: State
+        :return: state, schedulable_task_instances
+        :rtype: (State, list[airflow.models.TaskInstance])
```

Comment:
```suggestion
        :rtype: Tuple[State, List[airflow.models.TaskInstance])
```

Reply:
For sphinx docs it should be `tuple` and `list`.

Reply:
This is what the current version renders as:

![image](https://user-images.githubusercontent.com/34150/72477531-a6955900-37e7-11ea-8f9d-f333d195de04.png)

Reply:
It seemed to me that valid PEP types were supported. In that case, it probably should be `tuple[State, list[airflow.models.TaskInstance]]`

Reply:
Yup, that's what I'm changing it too. Link for reference http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists (at the end of that section, just before "Cross-referencing Python objects")